src/server/admin/server/convert1_8.go

```diff
@@ -387,7 +376,7 @@ func convert1_8Op(op *admin.Op1_8) (*admin.Op1_9, error) {
 			},
 		}, nil
 	default:
-		return nil, fmt.Errorf("Unrecognized 1.7 op type:\n%+v", op)
+		return nil, fmt.Errorf("unrecognized 1.7 op type:\n%+v", op)
```

Comment:
unrelated, but I think this is a copy-paste error and `1.7` should be `1.8` here?

Reply:
Good catch I'll fix it.